packages/mdc-list/index.js

```diff
@@ -51,25 +48,41 @@ class MDCList extends MDCComponent {
     return new MDCList(root);
   }
 
+  initialize() {
+    this.root_.querySelectorAll(MDCListFoundation.strings.LIST_ITEM_SELECTOR).forEach((listItemElement) => {
```

Comment:
This breaks in IE. `[].forEach.call(this.root_.querySelectorAll(...), (listItemElement) => ...)`